jaxrs/src/main/java/org/jboss/as/jaxrs/deployment/JaxrsIntegrationProcessor.java

```diff
@@ -175,6 +179,15 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
             setContextParameter(webdata, ResteasyContextParameters.RESTEASY_UNWRAPPED_EXCEPTIONS, "javax.ejb.EJBException");
         }
 
+        if (findContextParam(webdata, ResteasyContextParameters.RESTEASY_PREFER_JACKSON_OVER_JSONB) == null) {
+            final String prop = System.getProperty(ResteasyContextParameters.RESTEASY_PREFER_JACKSON_OVER_JSONB, null);
```

Comment:
This should probably use `WildFlySecurityManager.setPropertyPrivileged(ResteasyContextParameters.RESTEASY_PREFER_JACKSON_OVER_JSONB, null)`

Reply:
Fixed, thanks